### PR TITLE
Postgres Dackbox CVE SAC read tests

### PR DESCRIPTION
**Pull request by rhybrillou**
## Description

This change introduces the needed SAC changes to tests the image CVE and node CVE reads with scope checks.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
~~- [ ] Evaluated and added CHANGELOG entry if required~~
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

## Testing Performed

Added tests from the old dackbox, and ensured the can run against the postgres version

### CODE REVIEW

**Review by connorgorman — APPROVED** [review]
LGTM

File: tools/generate-helpers/pg-table-bindings/list.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func init() {
 		&storage.ComplianceRunResults{}:                         resources.Compliance,
```
Comment (md2119) [inline tools/generate-helpers/pg-table-bindings/list.go:31]: noticed that the node component and image component are not in the list.

**Review by md2119 — APPROVED** [review]

File: tools/generate-helpers/pg-table-bindings/testgraphtables/testg3grandchild1/store.go [hunk 2]
```diff
@@ -202,10 +202,13 @@ func (s *storeImpl) upsert(ctx context.Context, objs ...*storage.TestG3GrandChil
 			_, err := batchResults.Exec()
 			result = multierror.Append(result, err)
 		}
-		batchResults.Close()
+		closeErr := batchResults.Close()
 		if err := result.ErrorOrNil(); err != nil {
 			return err
 		}
+		if closeErr != nil {
```
Comment (connorgorman) [inline tools/generate-helpers/pg-table-bindings/testgraphtables/testg3grandchild1/store.go:209]: If would just do if err := batchResults.Close(); err != nil

I don't think we should try to look at result

File: central/deployment/store/postgres/full_store.go [hunk 3]
```diff
@@ -16,6 +16,7 @@ func NewFullStore(ctx context.Context, db *pgxpool.Pool) store.Store {
 	}
 }
 
+// FullStoreWrap augments the wrapped store with ListDeployment functions
```
Comment (dhaus67) [inline central/deployment/store/postgres/full_store.go:19]: super-nit:
```suggestion
// FullStoreWrap augments the wrapped store with ListDeployment functions.
```

**Review by dhaus67 — APPROVED** [review]

**Pull request merged**